app/Organizer/OrganizerServiceProvider.php

```diff
@@ -28,13 +30,22 @@ public function register(): void
     {
         $container = $this->getContainer();
 
+        // Use if you want to generate a JSON url endpoint
         $container->addShared(
             'organizer_iri_generator',
             fn () => new CallableIriGenerator(
                 fn ($cdbid) => $container->get('config')['url'] . '/organizers/' . $cdbid
             )
         );
 
+        // use if you want to sent the user to the user-friendly GUI version
+        $container->addShared(
+            self::ORGANIZER_FRONTEND_IRI_GENERATOR,
+            fn () => new CallableIriGenerator(
+                fn ($cdbid) => $container->get('config')['baseurl'] . '/organizers/' . $cdbid
```

Comment:
I think `baseurl` is too general if you want to point to the organizer inside the front end, maybe use something like `frontend_url`

Also I don't think the URL is correct, for example try
https://acc.uitdatabank.be/organizers/a9f9abc0-bb5e-4d03-8470-8e1a6f3a5f1a
which will fail, you need a `preview` path like
https://acc.uitdatabank.be/organizers/a9f9abc0-bb5e-4d03-8470-8e1a6f3a5f1a/preview

Reply:
Changed the name of the config value, and added the /preview suffix.